ui/src/app/base/components/ActionButton/ActionButton.js

```diff
@@ -89,7 +90,14 @@ const ActionButton = ({
       className={buttonClasses}
       disabled={disabled}
       ref={ref}
-      style={width ? { width: `${width}px` } : undefined}
+      style={
```

Comment:
This is a bit awkward tbh but it means that instead of the `<button>` having an empty style attribute it has no style attribute at all